BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -7,10 +7,11 @@
 //
 
 #import "UIImage+BOXBrowseSDKAdditions.h"
-#import <BoxContentSDK/BoxContentSDK.h>
-#import <BoxBrowseSDK/BoxBrowseSDK.h>
-#import "NSString+BOXBrowseSDKAdditions.h"
 #import "NSBundle+BOXBrowseSDKAdditions.h"
+#import <BoxBrowseSDK/BoxBrowseSDK.h>
+
+@import BoxContentSDK.BOXContentSDK;
```

Comment:
Not sure. Theoretically, this should import whatever is imported in the BOXContentSDK.h file. The categories are not in this file. 
I wonder what the best practice is : @import BoxContentSDK or @import BoxContentSDK.BOXContentSDK.

I'll try to read on this subject, in the meantime, fixed it.